department-of-veterans-affairs/caseflow
# Add request issues and docket number to Queue

Connects #5633 

### Description
Building out more of the queue details page.
1) Adding Docket Number
2) Adding Request Issues

Currently request issues are just freeform text, so we've move the `IssueListItem` component to `LegacyIssueListItem` and created `IssueListItem` to display free text request issues.

Docket numbers are based on this discussion: https://github.com/department-of-veterans-affairs/caseflow/issues/5533

![screen shot 2018-06-27 at 6 16 23 pm](https://user-images.githubusercontent.com/3885236/42002724-4d52b696-7a36-11e8-8286-539bf50d2578.png)


### Testing Plan
1. Make sure `:queue_beaam_appeals` is enabled.
2. Run `rake db:seed`
3. Go to `queue/beaam`
4. Click into an appeal and ensure the docket number and issues are properly displayed.
